Sources/SwiftGRPC/Runtime/ServiceServer.swift

```diff
@@ -23,39 +23,36 @@ open class ServiceServer {
   public let server: Server
 
   public var shouldLogRequests = true
-
+  
+  fileprivate let servicesByName: [String: ServiceProvider]
+  
   /// Create a server that accepts insecure connections.
-  public init(address: String) {
+  public init(address: String, services: [ServiceProvider]) {
     gRPC.initialize()
     self.address = address
     server = Server(address: address)
+    servicesByName = Dictionary(uniqueKeysWithValues: services.map { ($0.serviceName, $0) })
   }
 
   /// Create a server that accepts secure connections.
-  public init(address: String, certificateString: String, keyString: String) {
+  public init(address: String, certificateString: String, keyString: String, services: [ServiceProvider]) {
```

Comment:
How would you feel about changing "services:" to "serviceProviders:"? I think our "server", "service", and "provider" terminology can get confusing; more consistency could help here. (Maybe I'm still thinking like an Objective-C developer)

Reply:
(same comment for all three init methods)